docs/vc-http-api.yml

```diff
@@ -295,7 +301,8 @@ components:
           description: The JSON-LD type of the presentation.
         "holder":
           type: object
-          description: The holder
+          description: The holder - will be ignored if no proof is present since there is no proof of authority over the credentials
+          nullable: true
         "verifiableCredential":
```

Comment:
Why is this a 'verifiableCredential' -- shouldn't it be a 'verifiablePresentation' (with the ability to turn off the presentation proof check)?

Reply:
that is the array of credential(s) (on line 306) inside the `Presentation` object

Reply:
My bad... that's fine then.

Looking at this stuff feels like peering through a straw at times. 

The W3C PRs have a "Preview" mode where they generate the entire human-readable document and you can read through it w/o having to interpret the HTML in your head. I don't know if we'd be able to do the same here -- since this is taking YAML -> index.html.

Reply:
yeah, this stuff can be painful to mull though - i typically like to run through redoc, swagger, or similar on the cli othewise things get can get confusing.

for example:
```
~/devel/oss/vc-http-api$ redoc-cli serve docs/vc-http-api.yml 
{
  ssr: undefined,
  title: undefined,
  watch: undefined,
  templateFileName: undefined,
  templateOptions: {},
  redocOptions: {}
}

Server started: http://127.0.0.1:8080
```
will validate the yaml and serve it up local to make sure things are sane